UPGRADE-1.13.md

```diff
@@ -5,332 +5,372 @@
+1. Interface `Sylius\Bundle\ShopBundle\Calculator\OrderItemsSubtotalCalculatorInterface` and
+   class `Sylius\Bundle\ShopBundle\Twig\OrderItemsSubtotalExtension` responsible for the `sylius_order_items_subtotal`
+   twig function have been deprecated and will be removed in Sylius 2.0.
+   Use the `::getItemsSubtotal()` method from the `Order` class instead.
 
-1. Not passing an `$entityManager` and passing a `$doctrineRegistry` to `Sylius\Bundle\CoreBundle\Installer\Provider\DatabaseSetupCommandsProvider` constructor is deprecated and will be prohibited in Sylius 2.0.
+1. Interface `Sylius\Component\Core\Promotion\Updater\Rule\ProductAwareRuleUpdaterInterface` has been deprecated and
+   will be removed in Sylius 2.0.
 
-1. Product variants resolving has been refactored for better extendability.
-   The tag `sylius.product_variant_resolver.default` has been removed as it was never used.
+1. Both `getCreatedByGuest` and `setCreatedByGuest` methods were deprecated
+   on `Sylius\Component\Core\Model\OrderInterface`.
+   Please use `isCreatedByGuest` instead of the first one. The latter is a part of the `setCustomerWithAuthorization`
+   logic and should be used only this way.
 
-   All internal usages of service `sylius.product_variant_resolver.default` have been switched to `Sylius\Component\Product\Resolver\ProductVariantResolverInterface`, if you have been using the
-   `sylius.product_variant_resolver.default` service apply this change accordingly.
+1. The `Sylius\Bundle\ShippingBundle\Provider\Calendar` and `Sylius\Bundle\ShippingBundle\Provider\DateTimeProvider`
+   have been deprecated and will be removed in Sylius 2.0. Use `Symfony\Component\Clock\Clock` instead.
 
-1. Due to optimizations of the Order's grid the `Sylius\Component\Core\Repository\OrderRepositoryInterface::createSearchListQueryBuilder` method bas been deprecated in both the interface and the class, and replaced by `Sylius\Component\Core\Repository\OrderRepositoryInterface::createCriteriaAwareSearchListQueryBuilder`.
-   Also `Sylius\Component\Core\Repository\OrderRepositoryInterface::createByCustomerIdQueryBuilder` has been deprecated in both the interface and the class, and replaced by `Sylius\Component\Core\Repository\OrderRepositoryInterface::createByCustomerIdCriteriaAwareQueryBuilder` for the same reason. Both changes affect
-   `sylius_admin_order` and `sylius_admin_customer_order` grids configuration.
+1. Since catalog promotion action and scope validations have been rewritten to be more inline with symfony, the previous
+   abstraction has been deprecated. This includes:
+    - `Sylius\Bundle\PromotionBundle\Validator\CatalogPromotionAction\ActionValidatorInterface`
+    - `Sylius\Bundle\PromotionBundle\Validator\CatalogPromotionScope\ScopeValidatorInterface`
+    - `Sylius\Bundle\PromotionBundle\Validator\CatalogPromotionActionValidator`
+    - `Sylius\Bundle\PromotionBundle\Validator\CatalogPromotionScopeValidator`
```

Comment:
Maybe we should have a separate section for architectural changes?
This would probably also apply to points at lines 237, 262, 286, 335 and 347